docs/guides/external-node/00_quick_start.md

```diff
@@ -10,28 +10,28 @@ To start a mainnet instance, run:
 
 ```sh
 cd docker-compose-examples
-docker compose --file mainnet-external-node-docker-compose.yml up
+sudo docker compose --file mainnet-external-node-docker-compose.yml up
```

Comment:
pls don't add sudo to commands. It really depends on the setup on whether sudo is needed or not

Reply:
The default setup requires you to use sudo. If you change your setup to not require calling sudo, then you know that.